src/Components/Header/Header.jsx

```diff
@@ -122,25 +122,22 @@ function Header({ program, loadProgram, history, admin, curRoute, routes }) {
       programId: newProgram._id
     });
     loadProgram(newProgram._id);
-    if (!admin) {
-      history.push("/applications");
-    }
     //Load the application data into redux
   };
   const classes = useStyles();
   const validPrograms = allPrograms.isPending
     ? []
     : allPrograms.value.filter((p) => !!myProgramsMap[p._id]);
 
-  let hasAdminAccessForCurrentProgram = false;
-  if (programsMap[program]) {
-    const userProgram = appUser.programs.find((prog) => {
-      return programsMap[program]._id === prog.id;
-    });
-    if (userProgram && userProgram.role === "admin") {
-      hasAdminAccessForCurrentProgram = true;
-    }
-  }
+  const hasAdminAccessForCurrentProgram = true;
+  // if (programsMap[program]) {
+  //   const userProgram = appUser.programs.find((prog) => {
+  //     return programsMap[program]._id === prog.id;
+  //   });
+  //   if (userProgram && userProgram.role === "admin") {
+  //     hasAdminAccessForCurrentProgram = true;
+  //   }
+  // }
```

Comment:
Perhaps leave a comment or a TODO is there's future action to be done on this. Otherwise, remove.

Reply:
I added comment